### ISSUE TITLE
Allow the host to confirm reception of W9

### ISSUE THREAD

**Issue by xdamman** [opencollective/opencollective#1200]
Whenever the host pays an expense that has the comment from the W9 bot, we could automatically assume that the host has received the W9 (otherwise they wouldn't have paid the expense).

So when that happens, we could automatically add the UserId to the white list of users that shouldn't receive that notification anymore.

We could use the `Notifications` table to record that but since we want to record this at the host level and not the collective level, the `Notifications` table is not perfectly suited for that. So we can keep the white list attached to the `HostCollective`.

**Comment by piamancini** [comment]
- does the w9 bot differentiate between invoices and expenses? reimbursements are not taxable income, only invoices are. It seems like it's going to all.

also, the bot is sending the message to every host, we are not taking care of w9s for hosts other than the ones we manage. w9 emails from wwcode should go to wwcode not to us and it should be an opt-in feature for then in any case.

Sorry if this is not the correct issue, it the only one I found open on this.

**Comment by xdamman** [comment]
The issue is closed. This is first version of the bot to address the issue started by @alannallama https://github.com/opencollective/opencollective/issues/1155

It's a good first version but there are indeed many ways this could be improved. Let's create a new issue if more features are needed. I think the WWCode issue is probably the most pressing one as we don't want to receive W9 on their behalf. I'll create a new issue for that.
The receipt / invoice is a nice to have. It's better to be safe than sorry so it's ok to have false positive.

**Comment by piamancini** [comment]
I agree the receipt issue is secondary but I wouldn't treat it as a nice to have, it's confusing people since they are wondering if it's now part of their taxable income. I'd err on the side of being extra clear with these things as they tend to be a source of anxiety for many

**Comment by piamancini** [comment]
ok. How about this: 

> 
>   | The total amount of the expenses that you have submitted this year to this host exceeds $600. To comply with the IRS, we need you to send us by email the W9 form if you are a US resident (if you are not a US resident, please send the W-8BEN form for individuals or the W-8BEN-E form for companies) before we can proceed with this payment. More info on our wiki.
> 
> Note: if your expenses are reimbursements not invoices, please disregard this message.
> -- | --

**Comment by alannallama** [comment]
Let's be careful with the language. It doesn't matter if the person is a US *resident*, it matters if they are a US *person*, which includes citizens living abroad.

Once I get full access to the drop box folders with the forms, I will start manually uploading the forms as they come in. I plan to go through and rename all the files to the person's name, so I can scan the list when doing payments and see if we have one from a person. 

This is obviously a slow and error-prone process, so someday it would be great if there were a way to confirm receipt in the system.

**Comment by piamancini** [comment]
That is literally the language we now have, I just copy pasted. We should edit it if it's incorrect. Which I agree with you, it is. 
@xdamman You'll need to give @alannallama access to the dropbox, I don't have rights to add a new member.

**Comment by marcogbarcellos** [comment]
Do we have a final text here? I will deploy a new change in the codebase today and I could include any change in the text along with it. thanks in advance.

**Comment by piamancini** [comment]
The total amount of the expenses that you have submitted this year to this host exceeds $600. To comply with the IRS, we need you to send us by email the W9 form if you are a US person (a US citizen or US resident or a US incorporated entity) before we can proceed with this payment. If you are not a US person please send the W-8BEN form for individuals or the W-8BEN-E form for companies. More info on our wiki.

Note: if your expenses are reimbursements not invoices, please disregard this message.
-- | --

**Comment by marcogbarcellos** [comment]
cc @piamancini result in staging:
![screen shot 2018-08-13 at 4 39 42 pm](https://user-images.githubusercontent.com/8717041/44057101-2e235dd2-9f18-11e8-9ce8-ec138396256e.png)

**Comment by piamancini** [comment]
@marcogbarcellos it's hard to parse as is so I would do this instead: 

The total amount of the expenses that you have submitted this year to this host exceeds $600. To comply with the IRS, before we can proceed with this payment, we need you to [send us by email](mailto:w9@opencollective.com) one of the following tax forms: 

[W9](https://www.irs.gov/pub/irs-pdf/fw9.pdf) if you are a US person (a US citizen or US resident or a US incorporated entity) 
[W8-BEN](https://www.irs.gov/pub/irs-pdf/fw8ben.pdf) (for individuals) or [W8-BEN-E ](https://www.irs.gov/pub/irs-pdf/fw8bene.pdf)(for companies) who are not a US person
[More info on our wiki.](https://github.com/opencollective/opencollective/wiki/Submitting-Expenses#taxes)
Note: if your expenses are reimbursements not invoices, please disregard this message.

**Comment by marcogbarcellos** [comment]
no problems! will do that. thanks
